src/shapes/Circle.ts

```diff
@@ -41,12 +46,18 @@ export interface SerializedCircleProps
 
 export interface CircleProps extends FabricObjectProps, UniqueCircleProps {}
 
-const CIRCLE_PROPS = ['radius', 'startAngle', 'endAngle'] as const;
+const CIRCLE_PROPS = [
+  'radius',
+  'startAngle',
+  'endAngle',
+  'counterclockwise',
+] as const;
 
 export const circleDefaultValues: UniqueCircleProps = {
   radius: 0,
   startAngle: 0,
   endAngle: 360,
+  counterclockwise: true,
```

Comment:
Isn't this false by default?

Reply:
Can you add this to the appropriate test?

Reply:
Yes false by default if is counterClockwise